pipenv/vendor/vendor.txt

```diff
@@ -12,3 +14,4 @@ ruamel.yaml==0.18.6
 shellingham==1.5.4
 tomli==2.0.1
 tomlkit==0.12.4
+zipp==3.18.1
```

Comment:
The vendor file of pip has this nice convention that they indent packages which are depends.

e.g. if zipp is pulled in because of import-metadata, they put it underneath it with indentation.


```
importlib-metadata==6.5.1
   zipp=3.18.1
```
   
Shall we adopt this too? It will be clearer to track and remove vendored libraries as we remove old python versions.